### PR TITLE
feat: add google actions for doc, sheet, draw, gmail

### PR DESCRIPTION
**Pull request by 4www**
This PR adds the possibility of some new actions:
- `+draw [title]` opens a google draw with optional title
- `+doc [title]` google docs
- `+sheet` google spreadsheet (unfortunatelly no tite support)
- `+gmail` google mail new mail (no `subject`, or `to` or `body` fields, but maybe better)

At first I wanted to find a way to pass the body instead of the title (as a parameter), but it seems actually better like this, since the body is anyways too complex (and too provider specific) to manipulate.

https://mobile.twitter.com/googledocs/status/1055490445088903168

### CODE REVIEW

File: main.js [hunk 1]
```diff
@@ -126,6 +130,7 @@ var App = {
 	find(request) {
 		if(!request) return;
 		var url = this.decodeUserRequest(request);
+		console.log('opening url:', url)
```
Comment (oskarrough) [inline main.js]: remove?

**Review by oskarrough — APPROVED** [review]

File: readme.md [hunk 2]
```diff
@@ -84,7 +84,11 @@ But all the idea with `Find!` is to use the following `!keywords`
 
 | Action keyword  | site                                            |
 | ---             | ---                                             |
-| +r4             | radio4000.com add a new track from URL          |
+| +r4 [url]       | radio4000.com add a new track from URL          |
+| +draw [title]   | open a new drawing In Google Drive Draw         |
+| +doc [title]    | open a new Google Docs document                 |
+| +sheet          | open a new Google Spreadsheets document         |
```
Comment (oskarrough) [inline readme.md:90]: sheet also supports title no?

File: readme.md [hunk 3]
```diff
@@ -84,7 +84,11 @@ But all the idea with `Find!` is to use the following `!keywords`
 
 | Action keyword  | site                                            |
 | ---             | ---                                             |
-| +r4             | radio4000.com add a new track from URL          |
+| +r4 [url]       | radio4000.com add a new track from URL          |
+| +draw [title]   | open a new drawing In Google Drive Draw         |
```
Comment (oskarrough) [inline readme.md]: "In" -> "in"

File: main.js (hunk 1 above)
Comment (4www) [reply main.js]: si!

File: readme.md (hunk 2 above)
Comment (4www) [reply readme.md:90]: nope, it is added to the url to prevent that the parameter added by the user ends up concatenated in the url

Comment (4www) [reply readme.md:90]: it has to be fixed...

**Pull request merged**